.rubocop.yml

```diff
@@ -5,6 +5,13 @@ Style/FrozenStringLiteralComment:
 Lint/AmbiguousOperator:
   # buggy cop
   Enabled: false
+Layout/LineLength:
+  Max: 120
+Metrics/ClassLength:
+  Exclude:
+    - 'test/**/*_test.rb'
```

Comment:
Added this here to cover gems we create and their tests suites as well as Rails projects